### DISCUSSION TITLE
Does copilot store any kind of data for Copilot Individual

### DISCUSSION THREAD

**Discussion by swapnilpatwa-druck** [community/community#64718] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Does copilot store any kind of data for Copilot Individual Plan? I get that it won't store for business.

**Comment by iqbal-web** [comment]
https://docs.github.com/en/copilot/overview-of-github-copilot/about-github-copilot-for-individuals

**Comment by arafajs** [comment, marked as answer]
Yes, GitHub Copilot for Individuals does store some data. This data includes:

- **File content snippets:** This is the text of the files that you are editing when you use Copilot.
- **Suggestions**: This is the code that Copilot suggests to you.
- **Modifications to suggestions**: This is the code that you modify after Copilot suggests it.
- **Usage information**: This information includes things like the language you are using, the IDE or editor you are using, and the time and date of your interactions with Copilot.
- **More to read**: https://docs.github.com/en/copilot/overview-of-github-copilot/about-github-copilot-for-individuals

**Reply by SZahraM** [reply]
Hello, the link mentions that GitHub and Microsoft use User Engagement Data, Prompts, and Suggestions to enhance GitHub Copilot. Do you know if there is a way to access the logs of my own usage?
